### PR TITLE
 footer attempt 2

### PR DESCRIPTION
**Pull request by Ivan-M124**

### CODE REVIEW

File: footer attempt 2 [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<html>
+    <body style="background-color:lightsteelblue;">
```
Comment (carlosrymer) [inline footer attempt 2:2]: You have two body and html tags on the document. It should be one. The CSS is also not in a style block. It should be in a separate file.

File: footer attempt 2 [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    font-size: 1rem;
+}
+
+#footer-email-btn:hover {
+    cursor: pointer;
+    background-color: maroon;
+    transition: all 0.4s ease-out;
+}
+
+@media screen and (max-width: 1150px) {
```
Comment (carlosrymer) [inline footer attempt 2:130]: Curious, did you learn about media queries?

Comment (Ivan-M124) [reply footer attempt 2:130]: actually i followed along on a tutorial to have a base footer. and if anything i can try to find a work around or some other possible solution(s)

File: footer attempt 2 (hunk 1 above)
Comment (Ivan-M124) [reply footer attempt 2:2]: i will try to work on that now

File: footer attempt 2 (hunk 2 above)
Comment (carlosrymer) [reply footer attempt 2:130]: Got it. That's ok. I think you just need to move these into a separate CSS file and update the HTML.

File: footer attempt 2 (hunk 1 above)
Comment (carlosrymer) [reply footer attempt 2:2]: The CSS is still in this file.

**Pull request merged**